docs/contributing.md

```diff
@@ -0,0 +1,77 @@
+## Contributing
+
+### Development server
+
+When developing a package, to run the playground with your latest changes, cd to the package directory and then run `npm start`. For example, to run the playground, run:
+
+```bash
+cd packages/core
+npm install
+npm start
+```
+
+A live development server showcasing components with hot reload enabled will then run at [localhost:8080](http://localhost:8080).
+
+### Coding style
+
+All the JavaScript code in this project conforms to the [prettier](https://github.com/prettier/prettier) coding style. Code is automatically prettified upon commit using precommit hooks.
+
+### Documentation
+
+We use [mkdocs](https://www.mkdocs.org/) to build our documentation. To run documentation locally, run:
+
+```bash
+pip install mkdocs==1.0.4
+mkdocs serve
+```
+
+Documentation will be served on [localhost:8000](http://localhost:8000).
+
+### Tests
+
+```bash
+npm test
+```
+
+#### Code coverage
+
+Code coverage reports are currently available only for the `@rjsf/core` package. They are generated using [nyc](https://github.com/istanbuljs/nyc) each time the `npm test-coverage` script is run.
+The full report can be seen by opening `./coverage/lcov-report/index.html`.
+
+### Notes on the local playground
+
+The playground located in `packages/playground` contains code that is published as the `@rjsf/playground` package. This package is used by the local playgrounds in packages such as `packages/core`, so that when you run `npm start` from `packages/core`, it will run the playground component from `@rjsf/playground` but with the local changes to react-jsonschema-form.
+
+Note that when doing local development in a package such as `packages/core`, making changes to `packages/playground` will not affect the local playground that is run from `packages/core`. This should be fine in most cases for development, unless you need to update and test the default playground examples. In this case, you could use `npm link` to use the latest changes both from the playground and your package. As an example, to do this when developing locally on `packages/core`, you can run:
+
+```bash
+cd packages/playground
+npm run build:lib
+npm link
+cd ../core
+npm link @rjsf/playground
+npm start
+```
```

Comment:
Perhaps this should be done when the user pulls the package rather than an additional step otherwise this works for me 👍